src/scss/_mixins.scss

```diff
@@ -0,0 +1,50 @@
+////
+/// @group o-label
+/// @link http://registry.origami.ft.com/components/o-label
+////
+
+@mixin oLabelBase() {
+	.o-label {
+		@include oTypographySans;
+		font-size: _oLabelGet('default-font-size');
+		font-weight: 500;
```

Comment:
We don't usually set `font-weight` manually. Is it overriding something?

Reply:
No, I just can't find a way in o-typography to set font weight to use Metric Medium. May just not have looked hard enough?

Reply:
never mind I worked it out

Reply:
wait, no I didn't

Reply:
I don't think we have Metric Medium 🤔